trim.production.js

```diff
@@ -144,16 +258,27 @@ module.exports = function (apiUrl, apiToken, debug) {
   assert(typeof apiUrl === 'string', 'Argument 1 to instantiate the TRIM wrapper must be a valid url.')
   assert(typeof apiToken === 'string', 'Argument 2 to instantiate the TRIM wrapper must be a valid apiToken.')
 
-  url = apiUrl;
-  token = apiToken;
+  url = apiUrl
+  token = apiToken
   if (!!debug)
-    require('request-debug')(request);
+    require('request-debug')(request)
 
   return {
-    getContainer: getContainer,
+    PRIVACY_LEVELS: PRIVACY_LEVELS,
+    isValidPrivacyLevel: isValidPrivacyLevel,
     getDocument: getDocument,
-    createContainer: createContainer,
+
     createRecord: createRecord,
-    getPrivateContainer: getPrivateContainer
+    createPublicRecord: createPublicRecord,
+    createPrivateRecord: createPrivateRecord,
+
+    getContainer: getContainer,
+    getPublicContainer: getPublicContainer,
+    getPrivateContainer: getPrivateContainer,
+
+    createContainer: createContainer,
+    createPublicContainer: createPublicContainer,
+    createPrivateContainer: createPrivateContainer
```

Comment:
What version of node are we supporting with this module? I assume > 4.2
in which case you could drop all the boilerplate such as `getContainer: getContainer,` and just use `getContainer,`